careers/_template-students-and-graduates.html

```diff
@@ -0,0 +1,9 @@
+<h2 class="h4">Students & Recent Graduates</h2>
+<p class="short-desc">
+    Find opportunities in public service for a new generation of leaders.
+</p>
+<p class="short-desc">
+    <a href="/careers/students-and-graduates/" class="jump-link jump-link__right">
+        Learn about students & graduates opportunities
```

Comment:
@sarahsimpson09 Also, should this be `recent graduates` in the link as well? This is this bit:

![screen shot 2015-07-02 at 9 55 59 am](https://cloud.githubusercontent.com/assets/704760/8478626/93c3a934-20a0-11e5-8d7c-da0ffa7ab65d.png)